### DISCUSSION TITLE
Could not resolve host

### DISCUSSION THREAD

**Discussion by seelk07** [alexjustesen/speedtest-tracker#1498] [Q&A]
I updated to the latest version 0.20.3.  Since then I am no longer able to get successful speedtest runs.  I get **Could not resolve host** error.  Please note, both the VM and the container are able to ping google.com successfully.  Moreover, I am able to execute speedtest via the CLI of the container and that also works.  Please help!

**Comment by alexjustesen** [comment]
Check out #1484
